online-shop/src/app/services/products.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Product } from 'src/app/modules/shared/types/products.types';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductService {
+
+  apiBaseUrl = environment.apiUrl;
+
+  constructor(private http: HttpClient) {}
+
+  getAllProducts(): Observable<Product[]> {
+    return this.http.get<Product[]>(this.apiBaseUrl+'/products');
```

Comment:
I like the usage of string templates for the other requests, so you could use it here as well to be consistent.

Reply:
Done.